projects/plugins/crm/admin/settings/locale.page.php

```diff
@@ -362,6 +362,18 @@
 								?>
 								</select>
 								<p><?php echo wp_kses( sprintf( __( 'All Noto fonts are licensed under the <a href="%s" target="_blank">SIL Open Font License</a>.', 'zero-bs-crm' ) . ' ' . __( 'For more information, please go <a href="%s" target="_blank">here</a>.', 'zero-bs-crm' ), 'https://fonts.google.com/attribution', 'https://fonts.google.com/noto/specimen/Noto+Sans/about' ), $zbs->acceptable_restricted_html ); ?></p>
+								<p>
+									<?php
+										echo wp_kses(
+											sprintf(
+												/* translators: %s is the URL of the CRM notifications page. */
+												__( 'If a font does not install, please  <a href="%s" >check your notifications</a> for any error messages. Note that some larger font files may take up to a minute to download.', 'zero-bs-crm' ),
+												esc_url( get_site_url() . '/wp-admin/admin.php?page=zerobscrm-notifications' )
+											),
+											$zbs->acceptable_restricted_html
+										);
+										?>
+								</p>
```

Comment:
❤️

Reply:
```suggestion
								<p>
									<?php
										echo wp_kses(
											sprintf(
												/* translators: %s is the URL of the CRM notifications page. */
												__( 'If a font does not install, please <a href="%s" >check your notifications</a> for any error messages. Note that some larger font files may take up to a minute to download.', 'zero-bs-crm' ),
												esc_url( get_site_url() . '/wp-admin/admin.php?page=zerobscrm-notifications' )
											),
											$zbs->acceptable_restricted_html
										);
										?>
								</p>
```

Reply:
Might be worth linking to the KB article as well?

https://kb.jetpackcrm.com/knowledge-base/using-custom-fonts-in-crm-pdfs-e-g-invoice-templates/

Reply:
Great idea.